### PR TITLE
 Add template interpretation in config 

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/__init__.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def post(self, command, output_dir, vars_):
         """
 
         if os.name == 'posix':
-            for file_ in ("docker-run", "docker-compose-run"):
+            for file_ in ("docker-run", "docker-compose-run", "bin/eval-templates"):
                 dest = os.path.join(output_dir, "CONST_create_template", file_)
                 subprocess.check_call(["chmod", "+x", dest])
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/__init__.py]: Smells like déjà vu. Please refactor into a function.

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+find / -name *.tmpl -print | while read file
+do
+    envsubst < ${file} > ${file%.tmpl}
+done
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates]: Arn't you missing the `exec "$@"`?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+find / -name *.tmpl -print | while read file
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates]: Isn't that a bit slow to search at the root of the fs?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates (hunk 2 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates]: To do what? I don't have anything to exec ...

Comment (pvalsecc) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates]: It's not used as an entrypoint?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates]: Yes, but in the config container, than he hasn't any other "entry point"...

Comment (pvalsecc) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates]: By not putting an exec in there, you make the life of the dev wanting to debug this image harder... anyway, not a big deal...

**Pull request merged**